narratives/ncov_sit-rep_2020-04-24.md

```diff
@@ -0,0 +1,259 @@
+<!-- This is left-side text -->
+# [At least 8 introductions to Senegal, with 2 local transmission chains](https://nextstrain.org/ncov/africa/2020-04-23?d=tree,map&f_country=Senegal&f_region=Africa&m=div&p=full)
+
+Similar to the patterns found in the DRC, here we see samples from Senegal spanning the full genetic diversity of the tree. The phylogenetic placement of these samples indicates that there were at least 8 separate introductions to Senegal between late February and late March. Senegal reported its first  COVID-19 cases to the WHO on February 28, the same day that the first viral sample was collected for sequencing.
+<br><br>
+As we reported in our [April 3 Situation Report](https://nextstrain.org/narratives/ncov/sit-rep/2020-04-03?n=5), at least 2 of these introductions in late February or early March have led to ongoing local transmission clusters in Dakar and Tabou, respectively.
+
+
+<!-- There is no right-side text --><!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [At least 7 introductions to Ghana](https://nextstrain.org/ncov/africa/2020-04-23?d=tree,map&f_country=Ghana&f_region=Africa&p=full)
+
+There are 1,154 confirmed cases of COVID-19 in Ghana with 9 known deaths. On Nextstrain, there are 14 sequences from Ghana which are spread across the phylogenetic tree, representing at least 7 separate introductions.
+<br><br>
+The first case in Ghana was reported to the WHO on March 11, and the first viral genome on Nextstrain was collected on March 24.
+
+<!-- There is no right-side text --><!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Local transmission in Greater Accra, Ghana](https://nextstrain.org/ncov/africa/2020-04-23?label=clade:B4&c=division&d=tree,map&f_country=Ghana,Senegal&f_region=Africa&p=full&r=division&legend=closed)
+
+Here, we see a cluster of closely related cases from the Greater Accra region of Ghana, sampled between March 25 and March 30.
+The common ancestor of these cases dates to mid-February or early March, and was most likely imported from Asia.
+<br><br>
+Interestingly, there is a sample from Senegal that groups tightly with this cluster, suggesting the possibility of a transmission between Senegal and Ghana.
+
+<!-- There is no right-side text --><!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Scattered introductions to Algeria, Gambia, Nigeria, and South Africa](https://nextstrain.org/ncov/africa/2020-04-23?c=division&d=tree,map&f_country=Algeria,Gambia,Nigeria,South%20Africa&f_region=Africa&p=full)
+
+Here, we see samples isolated from Algeria, Gambia, Nigeria and South Africa. These samples are scatted across the full tree, and are largely isolated from one another.
+<br><br>
+This is evidence for many separate introductions, largely from Europe, to many nations in Africa. Without more data, we can't yet say which -- if any -- of these sparks resulted into local outbreaks.
```

Comment:
```suggestion
This is evidence for many separate introductions, largely from Europe (keeping in mind possible sampling bias due to the large number of European samples), to many nations in Africa. Without more data, we can't yet say which -- if any -- of these sparks resulted into local outbreaks.
```
I think we just need to be really clear about the fact that bias could be influencing that these all look to come from Europe! :)